finalfrontier-utils/src/util.rs

```diff
@@ -452,21 +460,32 @@ pub enum VocabConfig {
     SimpleVocab(SimpleVocabConfig),
 }
 
-/// Construct `SubwordVocabConfig` from `matches`.
+/// Construct `SubwordVocabConfig` or `SimpleVocabConfig` from `matches`.
 fn vocab_config_from_matches(matches: &ArgMatches) -> VocabConfig {
     let discard_threshold = matches
         .value_of(DISCARD)
         .map(|v| v.parse().or_exit("Cannot parse discard threshold", 1))
         .unwrap();
-    let min_count = matches
-        .value_of(MINCOUNT)
-        .map(|v| v.parse().or_exit("Cannot parse mincount", 1))
-        .unwrap();
     if matches.is_present(NO_SUBWORDS) {
-        VocabConfig::SimpleVocab(SimpleVocabConfig {
-            min_count,
-            discard_threshold,
-        })
+        if matches.is_present(VOCAB_SIZE) {
```

Comment:
This block occurs here and below (for `SubwordVocab`). You can have this block only once, assign the `VocabCutoff` result to a `vocab_cutoff` variable and use this variable in the simple/subword vocabs.